src/components/CopyValue/CopyValue.module.css

```diff
@@ -10,12 +10,14 @@
   composes: line;
 
   --copy-value-icon: url('../images/kilt-coin-circle.svg');
+  --icon-size: 1rem;
```

Comment:
When using the CSS variables we should exercise caution: their names are not scoped to the file. So if --icon-size is defined anywhere else in the  app and cascades to this line, it might have effect. Before scoping we used namespaces to solve this issue, and still need to use it for CSS variables. Please add a namespace like the previous variable.